Sources/Remote Logging/Crash Logging/CrashLoggingDataProvider.swift

```diff
@@ -10,6 +10,7 @@ public protocol CrashLoggingDataProvider {
     var buildType: String { get }
     var currentUser: TracksUser? { get }
     var additionalUserData: [String: Any] { get }
+    var eventSamplingRate: Double { get }
```

Comment:
In the diff above, this property maps to Sentry's `sampleRate`. [The docs](https://docs.sentry.io/platforms/apple/guides/ios/configuration/sampling/#sampling-error-events) refer to that property in the context of "sampling error events".

What do you think of renaming it `errorEventsSamplingRate`?

Reply:
I assume Sentry would clamp this value between 0 and 1 internally, but what do you think of doing the same on our end, similar to what we do fro performance tracking:

https://github.com/Automattic/Automattic-Tracks-iOS/blob/d883cd1720ca49c2f2ec7e8473c1f49eb0df0c4b/Tests/Tests/PerformanceTrackingTests.swift#L11-L31